app/javascript/app/assets/icons/attained.svg

```diff
@@ -0,0 +1 @@
+<svg width="50" height="60" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><defs><path id="a" d="M0 0h59.53v49.113H0z"/></defs><g fill="none" fill-rule="evenodd"><g transform="matrix(0 1 1 0 .5 0)"><mask id="b" fill="#fff"><use xlink:href="#a"/></mask><path d="M12.879 13.832c-.41.841-.87 1.663-1.426 2.41C6.233 23.257 0 23.257 0 23.257v3.974c5.658 0 9.315 3.528 11.361 5.95.803.951 1.445 2.023 2.041 3.116 4.164 7.634 12.262 12.816 21.571 12.816 14.086 0 25.403-11.862 24.507-26.141C58.717 10.825 48.89.917 36.75.062 26.289-.672 17.102 5.15 12.879 13.833z" fill="#113750" mask="url(#b)"/></g><path fill="#FFF" fill-rule="nonzero" d="M10.5 35.596l3.233-4.153 8.64 7.124L36.567 23l3.932 3.525L22.923 45.94z"/></g></svg>
```

Comment:
Icons are displaying the hardcoded `fill` colour `#113750` but instead they should be filled with  `#0a1941`.
Just the `launched` icon has `#0a1941` as `fill` attribute.
Would be nice to be able to style them with the `statusIcon` class.